src/components/custombutton-large.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+interface CustomButtonProps {
+  onClick: React.MouseEventHandler<HTMLButtonElement>;
+  text: string;
+}
+
+const CustomButton: React.FC<CustomButtonProps> = ({ onClick, text }) => {
+  return (
+    <button
+      type="submit"
+      onClick={onClick}
+      className="w-full cursor-pointer rounded-md bg-[#098D85] py-2 font-bold text-white"
```

Comment:
the tailwind colours have been updated in `main`.
so the new className for the colour should be `bg-primary-600`.